intrpvar.h

```diff
@@ -938,6 +938,10 @@ PERLVARI(I, InBitmap,	SV *, NULL)
 PERLVAR(I, CCC_non0_non230,	SV *)
 PERLVAR(I, Private_Use,	SV *)
 
+#ifdef HAS_MBRLEN
+PERLVAR(I, mbrlen_ps, mbstate_t)
+#endif
```

Comment:
You've define `mbrlen_ps` here and initialized it for a new interpreter in init_i18nl10n(), but it also needs to be zeroed or copied in perl_clone_using() in `sv.c`.

non-DEBUGGING perls will zero it, but DEBUGGING perls poison the new structure.